src/liblx/xml/sbmlMemoryStubs.h

```diff
@@ -9,12 +9,19 @@
 #define SBMLMEMORYSTUBS_H
 
 #include <ctype.h>
+#include <cstddef>
 
 extern "C"
 char* safe_strdup(const char* s);
 extern "C"
 void* safe_malloc (size_t size);
 
+/**
+ * Easier-to-read and NULL-friendly string comparison.
+ */
```

Comment:
This is duplicating some of my recent PR (although admittedly still under review)
Also, does it need to be declared as `extern "C"`?

Reply:
this whole file needs changing, see the exchange today. it should use the begin_c_decls and namespaces for everything. But i can add it ther.